doc/user/content/ops/monitoring.md

```diff
@@ -140,6 +146,17 @@ Materialize broadly publishes the following types of data there:
   averaged over 10 second windows.
 - Standard process metrics with a `process_*` prefix. For exmple, `process_cpu`.
 
+## System Status Views
+
+{{< warning >}}
+The `mz_catalog` views not part of Materialize's stable interface.
+Backwards-incompatible changes to the exposed views may be made at any time.
+{{< /warning >}}
+
+The views in `mz_catalog` provide a variety of ways to introspect materialize. An introduction to this catalog is available at our [Diagnostic Queries][] document.
+
+[Diagnostic Queries]: https://github.com/MaterializeInc/materialize/blob/main/doc/developer/diagnostic-queries.md
```

Comment:
We actually already have the beginnings of docs for `mz_catalog` here: https://materialize.com/docs/sql/system-catalog/. Would be good to link to that here. 

On a meta note I don't love the idea of linking to developer docs in the user docs. I think that's a good sign that the diagnostic queries doc needs to be promoted to a user doc. I'll add this as a work item for myself.

Reply:
I'm also going to add reference docs for all of the missing tables later today.

Reply:
I do think promoting that doc to user-doc would be useful. Having both reference documentation and guides for how to think about it is useful.

Reply:
I also think it'd be useful to link to both from this paragraph, how would you feel about me leaving in the current link, and you fixing it up with your follow-up PR? I agree with the principle of not linking to dev docs from user docs.